projects/addon-doc/components/api/api-item.component.ts

```diff
@@ -48,13 +50,15 @@ const SERIALIZED_SUFFIX = '$';
     ],
     templateUrl: './api-item.template.html',
     styleUrls: ['./api-item.style.less'],
+    exportAs: 'docAPIItem',
```

Comment:
It's unnecessary since it's a component, it would grab it by default anyway.

Reply:
Oh, yes, you're right
Removed